packages/aikit-respond-io/src/webhook/utils.ts

```diff
@@ -16,3 +19,51 @@ export function verifySignature(
     throw new SignatureVerificationError("Signature mismatch.");
   }
 }
+
+/**
+ * Converts a Respond.io message content to a Vercel AI SDK ModelMessage.
+ *
+ * @param messageContent The content of the message from Respond.io.
+ * @param role The role of the message sender.
+ * @returns ModelMessage.
+ */
+export function toAiModelMessage(
+  messageContent: MessageContent,
+  role: "user" | "assistant",
+): ModelMessage {
+  if (role === "user") {
+    const parts: UserContent = [];
+
+    if (messageContent.type === "text") {
+      parts.push({ type: "text", text: messageContent.text });
+    } else if (messageContent.type === "attachment") {
+      const attachment = messageContent.attachment;
+      if (attachment.type === "image") {
+        parts.push({
+          type: "image",
+          image: new URL(attachment.url),
+        });
+      } else {
+        throw new Error(`Unsupported attachment type: ${attachment.type}`);
```

Comment:
Are there potential other attachment types that we need to support in the future? If yes, please mark with // FUTURE tag and elaborate in error message that is is a temporary limitation. Users should reach out to the team to add it. Same for the checks later in the code.